test-on-pr.yaml

```diff
@@ -0,0 +1,90 @@
+name: Test on Pull Request 
+
+# Controls when the action will run. 
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'design/**'
+      - 'examples/**'
+      - 'extensions/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout autotune
+        uses: actions/checkout@v2
+        with:
+          repository: kruize/autotune
+          ref: integration-channel-RM-EM
+          path: autotune
+
+      - name: Checkout benchmarks
+        uses: actions/checkout@v2
+        with:
+          repository: kruize/benchmarks
+          path: benchmarks
+
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.3.0
+        with:
+          minikube version: 'v1.16.0'
+          kubernetes version: 'v1.19.2'
+      - name: Start minikube
+        run: |
+          kubectl cluster-info
+          kubectl get pods -n kube-system
+
+      - name: Build autotune
+        run: |
+          echo Build autotune
+          cd autotune
+          ./build.sh -t autotune:e2e
```

Comment:
I would prefer if we have a better name than `e2e`, maybe `integration-test` or `int-test` for short.

Reply:
I have changed it to int-test